common/src/test/java/com/tc/net/core/TCConnectionImplTest.java

```diff
@@ -132,7 +140,8 @@ public void testWriteEndsWhenClose() throws Exception {
         conn.putMessage(msg);
 
         new Thread(() -> sleepThenClose(conn)).start();
-        int w = conn.doWrite();
+        Thread.sleep(4000);
```

Comment:
Why the `sleep`?

Reply:
It's a bad test...updated the test to be more straightforward